### DISCUSSION TITLE
Adding a solver for mixed-int linear prog.

### DISCUSSION THREAD

**Discussion by zoharl3** [yalmip/YALMIP#967] [Possible bug or missing functionality] [answered]
I have an MILP, and I use gurobi, which I think is the best available option to solve it.

There is the comiso library

https://www.graphics.rwth-aachen.de/software/comiso/

which does something super naive:
- Solve the problem treating all vars as continuous.
- Pick an int var.
- Round it to the closest int, and keep its value fixed for the next iterations.
- Iterate until all int vars are fixed.

Somehow, this naive method beats gurobi by something like seconds vs minutes. Yes, it doesn't have any guarantees, and sometimes it fails (it doesn't go over all the tree like gurobi), but for my problem it's enough.

Can you perhaps suggest a way to extend yalmip to support comiso?
More specifically, I need the matrices of the problem. It's probably related to what I asked in the past, which wasn't possible: use yalmip for symbolic differentiation because scripting with sdpvar is convenient.

**Comment by johanlofberg** [comment, marked as answer]
Too small user-base so it is not relevant to add this solver. Code is currently going in oppositite direction with removal of old effectively small/exotic/unmaintained solvers.

Additionally, the solver appears to implement something which would be trivial to implement in high-level YALMIP code

If you simply need the numerical data of a model, export YALMIP model to some simple format (quadprog for instance)

**Comment by zoharl3** [comment]
Oh, so there is an export option to something readable. Somehow, I recalled that it's too low level to be useful, but it seems good enough.
Thanks
